std/encoding.d

```diff
@@ -2373,12 +2373,12 @@ abstract class EncodingScheme
         /**
          * Returns the standard name of the encoding scheme
          */
-        abstract override string toString();
+        abstract override string toString() @safe pure nothrow @nogc;
```

Comment:
Wouldn't this require that all classes that inherit from this also have their `toString` method `@safe pure nothrow @nogc`? That would be a backwards compatibility breaking change.